layers/lispy/packages.el

```diff
@@ -0,0 +1,67 @@
+;;; packages.el --- lispy layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2016 Sylvain Benner & Contributors
+;;
+;; Author: Justin Burkett <justin@burkett.cc>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+;;; Commentary:
+
+;;; Code:
+
+(defconst lispy-packages
+  '(lispy))
+
+(defun lispy/init-lispy ()
+  (use-package lispy
+    :defer t
+    :init
+    (progn
+      (defun spacemacs/lispy-off ()
+        (when lispy-mode
+          (lispy-mode -1)))
+      (defvar-local spacemacs-lispy-mode nil)
+      (defun spacemacs/toggle-lispy ()
+        "Toggle `lispy-mode'. If `holy-mode' is enabled, this is
+just a simple toggle. For vim and hybrid style (depending on the
+value of `hybrid-mode') this adds or removes hooks that turn on
+`lispy-mode' only in hybrid/insert state and only in the current
+buffer."
+        (interactive)
+        (pcase (list (bound-and-true-p holy-mode)
+                     (bound-and-true-p hybrid-mode)
+                     spacemacs-lispy-mode)
+          (`(t ,x ,y)
```

Comment:
`` `(t ,_ ,_)`` matches the same as `` `(t ,x ,y)`` but doesn't bind `x` and `y`.